glide-core/src/client/value_conversion.rs

```diff
@@ -5,8 +5,8 @@ use redis::{
     cluster_routing::Routable, from_owned_redis_value, Cmd, ErrorKind, RedisResult, Value,
 };
 
-#[derive(Clone, Copy)]
-pub(crate) enum ExpectedReturnType {
+#[derive(Clone, Copy, Debug)]
```

Comment:
I'm not sure we should add this to release code

Reply:
https://github.com/aws/glide-for-redis/pull/1487/files#diff-eaea3b88c569ecab22aeb7db41a9dc796aedb808b65d7f3ceceacc551b8a35c0R43
It generates code for debug string formatting (`{:?}`) we use for error messages. And I think if we don't use it, the code is dropped by optimizer.